library/Zend/Barcode/Renderer/Image.php

```diff
@@ -345,7 +345,9 @@ protected function _checkDimensions()
     public function render()
     {
         $this->draw();
-        header("Content-Type: image/" . $this->_imageType);
+        // PCR360-11102: The Content-Type header is breaking HTML barcodes, causing the iframe content to render with
```

Comment:
use @see tag for referencing Jira issues

Reply:
- [ ] Fix comment